yt/yt/server/lib/misc/job_reporter.cpp

```diff
@@ -74,7 +75,35 @@ class TJobRowlet
 
     size_t EstimateSize() const override
     {
-        return Report_.EstimateSize();
+        return ::NYT::EstimateSizes(
+            Report_.OperationId().Underlying(),
+            Report_.JobId().Underlying(),
+            Report_.State(),
+            Report_.StartTime(),
+            Report_.FinishTime(),
+            /*updateTime*/ TInstant::Now().MicroSeconds(),
```

Comment:
Why `MicroSeconds`?

Reply:
For simplicity, same as value used for `.UpdateTime` below.